gpiozero/ep.py

```diff
@@ -0,0 +1,25 @@
+"""
+Provides access to the gpiozero entry points:
+
+.. code-block:: python
+    from gpiozero.ep import PinFactory_entry_points
```

Comment:
Do we want a more meaningful name than `gpiozero.ep` ? :shrug:

Reply:
Any ideas what? I leant on the approach used for `exc` and tried to avoid anything that would collide with `entry_points`

Would it make sense to expose `PinFactory_entry_points` and `MockPinClass_entry_points` via `__init__.py`? In that case I would also add them to the docs; the question is "where?": a separate "Entry Points API" would be very short ...

Reply:
Again, a question for Ben & Dave...

Reply:
@bennuttall , @waveform80 : Would you like:

- anything exposing via __init__.py to make it easier for others to use as well
- some documentation on this topic (in which case where?)
- improved naming for the module?